### PR TITLE
Release 9.4.0

### PR DESCRIPTION
**Pull request by MarcinOrlowski**

### CODE REVIEW

File: .github/workflows/phpstan.yml [hunk 1]
```diff
@@ -56,4 +52,4 @@ jobs:
         with:
           configuration: "phpstan.neon"
           php_version: "${{ matrix.php }}"
-          memory_limit: 16M
+          # memory_limit: "16M"
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:55]: `memory_limit: 256M` :)

Comment (MarcinOrlowski) [reply .github/workflows/phpstan.yml:55]: IIRC i tried bumping it up, but higher values did not work (like were rejected).

**Comment by szepeviktor** [comment]
@MarcinOrlowski Try running PHPStan locally without `php-actions/phpstan` action.
It will work.

**Comment by szepeviktor** [comment]
If you wish I can contribute my default CI.

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -30,14 +30,13 @@ jobs:
       matrix:
         # quotes are needed it is treated as a number and zero at decimal part is gone at runtime
         laravel: ["9.x"]
-        # PHP 8.0 should be covered as well, but I switched to PHPUnit 10 and that requires PHP 8.1
-        php: ["8.1"]
+        php: ["8,0", "8.1"]
```
Comment (szepeviktor) [inline .github/workflows/phpunit.yml:33]: @MarcinOrlowski There is a comma here 👆🏻

Comment (MarcinOrlowski) [reply .github/workflows/phpunit.yml]: Damn. Thanks! 👏🏻 Things like this were the reason I had my previous IDE configured so it highlighted dots differently than commas.

**Pull request merged**